src/build.py

```diff
@@ -267,12 +267,13 @@ def JavaBinDir():
 NPROC = multiprocessing.cpu_count()
 
 
-if IsMac():
+if IsMac() or True:
```

Comment:
Does `or True` mean we are gonna run this in every platform?

Reply:
Oops, that was for local debugging.